script.js

```diff
@@ -113,6 +113,11 @@ const createRows =(data)=>{
     //adding event to search button
     searchbutton.addEventListener("click",()=>{
     const searchword= searchText.value.toLowerCase(); //.value is used retrieves the current text entered in the input field.
+    if (!searchword) {
+        alert("Please enter a search term.");
```

Comment:
if the search input is empty, then by default First Page should be shown,  after alert the list of users must be shown.

Reply:
resolved